### PR TITLE
#649293 - Identify external messages in blip cards

### PR DESCRIPTION
**Pull request by Plezito**
There is a new scenario where, besides the bot/desk and the user, a third party could join the conversation.

To differentiate if the message came from the bot/desk or the other user we created a new flag to be shown together with the send date.

We had to also change the card grouping rule, to not group cards sent from different senders.

And we refactored some card types to use the same date/external message component.

### CODE REVIEW

**Review by sallesweb — APPROVED** [review]

**Review by VMarchena — APPROVED** [review]

**Review by paulaTake — APPROVED** [review]

File: src/components/BlipGroupCard.vue [hunk 1]
```diff
@@ -165,23 +159,26 @@ export default {
...
       }
       for (let i = 1; i < this.documents.length; i++) {
-        const message = this.documents[i]
-        if (this.compareMessages(group.msgs[group.msgs.length - 1], message)) {
-          group.msgs.push(message)
-          group.date = message.date
-          group.status = message.status
-          group.reason = message.reason
+        const message1 = group.msgs[group.msgs.length - 1]
+        const message2 = this.documents[i]
```
Comment (nakahara-luiz) [inline src/components/BlipGroupCard.vue:163]: Is it possible to change this message1 / 2 a better name? like currentMessage, lastMessage?

Comment (Plezito) [reply src/components/BlipGroupCard.vue:163]: It is! I was just following the names given in the 'compareMessages' method. I'll change them right away.

**Review by nakahara-luiz — APPROVED** [review]

File: src/components/BlipGroupCard.vue (hunk 1 above)
Comment (Plezito) [reply src/components/BlipGroupCard.vue]: Done! I changed the group.msgs[group.msgs.length - 1] to lastMessage to reflect its position, and the this.documents[i] to currentMessage to indicate that it is the one we are currently iterating.

**Review by AndreSantosTake — APPROVED** [review]

**Review by leonardo-avila — APPROVED** [review]

**Pull request merged**